master/getting-started/kubernetes/flannel/migration-from-flannel.md

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
@bmckercher123 I think you should delete this doc and just use the existing one  - the link on the sidebar is pointing to the existing one anyway. That should be enough to get this merged I think.

Reply:
Fixed